WcaOnRails/app/models/post.rb

```diff
@@ -2,11 +2,37 @@
 
 class Post < ApplicationRecord
   belongs_to :author, class_name: "User"
+  has_many :post_tags, autosave: true, dependent: :destroy
 
   validates :title, presence: true, uniqueness: true
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  def reload
+    @tags = nil
+    super
+  end
+
+  attr_writer :tags
+
+  def tags
+    @tags ||= post_tags.pluck(:tag).join(",")
```

Comment:
Yeah, those are reasonable concerns. Though the "depreciation" of controller test isn't much to worry about.

Having read a bit more, I now lean towards that `accepts_nested_attributes_for` is the Railsy way to do this.

Like you said this works, there are tests, and it's not a very important part of the system, so if you want to merge it as is, that's fine by me. It would be fair for me to fix my complaint, and I'd like to, but I can't promise I'll get around to it.